### PR TITLE
Feature json validation

### PR DESCRIPTION
**Pull request by ecogodi**

### CODE REVIEW

File: test/karma.conf.js [hunk 1]
```diff
@@ -37,17 +37,23 @@ module.exports = (config) => {
...
       {pattern: 'node_modules/@zxing/library/umd/index.min.js', included: false, watched: false},
 
+      {pattern: 'node_modules/ajv/dist/*', included: false, watched: false},
       {pattern: 'node_modules/chart.js/Chart.umd.js', included: false, watched: false},
       {pattern: 'node_modules/chart.piecelabel.js/build/Chart.PieceLabel.min.js', included: false, watched: false},
       {pattern: 'node_modules/css-element-queries/css-element-queries.umd.js', included: false, watched: false},
       {pattern: 'node_modules/date-fns/date-fns.umd.js', included: false, watched: false},
       {pattern: 'node_modules/debug/debug.umd.js', included: false, watched: false},
       {pattern: 'node_modules/esprima/esprima.umd.js', included: false, watched: false},
+      {pattern: 'node_modules/ionic-selectable/bundles/ionic-selectable.umd.js', included: false, watched: false},
```
Comment (trik) [inline test/karma.conf.js]: remove this, unused

File: test/karma.conf.js [hunk 2]
```diff
@@ -37,17 +37,23 @@ module.exports = (config) => {
...
+      {pattern: 'node_modules/ionicons/**/*', included: false, watched: false},
```
Comment (trik) [inline test/karma.conf.js]: remove this, unused

File: test/karma.conf.js [hunk 3]
```diff
@@ -37,17 +37,23 @@ module.exports = (config) => {
...
       {pattern: 'node_modules/leaflet/leaflet.umd.js', included: false, watched: false},
+      {pattern: 'node_modules/ms/index.js', included: false, watched: false},
```
Comment (trik) [inline test/karma.conf.js]: remove this, unused

File: test/karma.conf.js [hunk 4]
```diff
@@ -37,17 +37,23 @@ module.exports = (config) => {
...
       {pattern: 'node_modules/moment/min/moment.min.js', included: false, watched: false},
       {pattern: 'node_modules/numeral/numeral.umd.js', included: false, watched: false},
       {pattern: 'node_modules/rxjs/**/*', included: false, watched: false},
 
+      {pattern: 'bundles/*.js', included: false, watched: false},
```
Comment (trik) [inline test/karma.conf.js]: remove this, unused

File: src/dev-app-mat/forms/forms-demo.ts [hunk 5]
```diff
@@ -22,11 +22,11 @@
 
 import {AjfForm, AjfFormSerializer} from '@ajf/core/forms';
 import {AjfContext} from '@ajf/core/models';
+import {AjfJsonValidator} from '@ajf/core/json-validation';
```
Comment (trik) [inline src/dev-app-mat/forms/forms-demo.ts]: please keep imports/export in alphabetical order

**Comment by ecogodi** [comment]
The last two commits address the change request

**Pull request closed without merging**